code/modules/detective_work/scanner.dm

```diff
@@ -88,6 +88,19 @@
 	else
 		to_chat(usr, "<span class='notice'>The scanner has no logs or is in use.</span>")
 
+/obj/item/detective_scanner/verb/clear_scanner()
+	set name = "Clear Scanner Records"
+	set category = "Object"
+	set desc = "Clear the records on your Forensic Scanner."
+
+	if(length(log) && !scanning)
+		log = list()
+		playsound(loc, 'sound/machines/ding.ogg', 50)
+		spawn(15) //So that it clears on the 'ding'
+			to_chat(usr, "<span class='notice'>Scanner logs cleared.</span>")
```

Comment:
Timers are preferred over spawns:
```suggestion
		addtimer(CALLBACK(GLOBAL_PROC, .proc/to_chat, usr, "<span class='notice'>Scanner logs cleared.</span>"), 1.5 SECONDS)  //So that it clears on the 'ding'
```

Reply:
If you want, yeah, go for it.